solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -50,6 +50,16 @@ export default {
             required: false,
             default: [],
```

Comment:
I know it's been this way for a while but `sections` and `subparts` are throwing warnings in Storybook:

```
[Vue warn]: Invalid default value for prop "subparts": Props with type Object/Array must use a factory function to return the default value.
```

default value should look like this to remove the warning:'

```
   subparts: {
        type: Array,
        required: false,
        default() {
            return []
        }
    }
```